test/cli_update_test.go

```diff
@@ -0,0 +1,93 @@
+package main
+
+import (
+	"strings"
+
+	"github.com/alibaba/pouch/test/command"
+	"github.com/alibaba/pouch/test/environment"
+
+	"github.com/go-check/check"
+	"github.com/gotestyourself/gotestyourself/icmd"
+)
+
+// PouchUpdateSuite is the test suite for help CLI.
+type PouchUpdateSuite struct{}
+
+func init() {
+	check.Suite(&PouchUpdateSuite{})
+}
+
+// SetUpSuite does common setup in the beginning of each test suite.
+func (suite *PouchUpdateSuite) SetUpSuite(c *check.C) {
+	SkipIfFalse(c, environment.IsLinux)
+
+	environment.PruneAllContainers(apiClient)
+
+	command.PouchRun("pull", busyboxImage).Assert(c, icmd.Success)
+}
+
+// TearDownTest does cleanup work in the end of each test.
+func (suite *PouchUpdateSuite) TearDownTest(c *check.C) {
+}
+
+// TestUpdateMemory is to verify the correctness of updating container memory.
+func (suite *PouchUpdateSuite) TestUpdateMemory(c *check.C) {
+	//TODO
+}
+
+// TestUpdateRunningContainer is to verify the correctness of updating a running container.
+func (suite *PouchUpdateSuite) TestUpdateRunningContainer(c *check.C) {
+	name := "update-running-container"
+
+	command.PouchRun("run", "-d", "-m", "300M", "--name", name, busyboxImage).Assert(c, icmd.Success)
+
+	command.PouchRun("update", "-m", "500M", name).Assert(c, icmd.Success)
+
+	file := "/sys/fs/cgroup/memory/memory.limit_in_bytes"
```

Comment:
I am afraid the test case is running on the host rather than inside container, then the path will not exist. @HusterWan